cypress/integration/plugins/gantt-chart-dashboards/gantt_ui.spec.js

```diff
@@ -54,103 +55,100 @@ describe('Dump test data', () => {
   });
 });
 
-describe('Save a gantt chart', () => {
+describe('Save a gantt chart', { defaultCommandTimeout: 20000 }, () => {
   beforeEach(() => {
+    CURRENT_TENANT.newTenant = 'private';
```

Comment:
Any reason we switch to a private tenant?

Reply:
Switching to private tenant since some tests are flaky due to tennant popup. For some reason, setting it to `global` resulted in our target index not being found.
![](https://ci.opensearch.org/ci/dbc/integ-test-opensearch-dashboards/2.13.0/7548/linux/x64/deb/test-results/5753/integ-test/ganttChartDashboards/with-security/cypress-screenshots/plugins/gantt-chart-dashboards/gantt_ui.spec.js/Save%20a%20gantt%20chart%20--%20Creates%20and%20saves%20a%20gantt%20chart%20(failed).png)

Reply:
Hmm, switched over to `global` but it looks like the popup still stays there on the first test `Create and saves a gantt chart`. @SuZhou-Joe, any ideas on why the popup still shows up even after setting the tenant, and only on the first test?

Reply:
Not familiar with the tenant popup either. @kavilla Do you know how to hide the tenant popup?